rustybeer/src/yeast/yeast.rs

```diff
@@ -0,0 +1,112 @@
+use csv::ReaderBuilder;
+use measurements::temperature::Temperature;
+//use regex::Regex;
```

Comment:
Unnecessary comment

Reply:
Sorry, will take it into use with the missing deserializations